ArduCopter/precision_landing.cpp

```diff
@@ -8,7 +8,7 @@
 
 void Copter::init_precland()
 {
-    copter.precland.init(400);
+    copter.precland.init(scheduler.get_loop_rate_hz());

```

Comment:
Gah!  This actually has to be exactly the same number as in the scheduler table.  I apologise for misleading you here.  Please revert just this patch.

Reply:
done